app/components/primer/alpha/select_panel_element.ts

```diff
@@ -314,11 +314,7 @@ export class SelectPanelElement extends HTMLElement {
         const itemContent = this.#getItemContent(item)
         if (!itemContent) continue
 
-        if (!this.isItemHidden(item) && !setZeroTabIndex) {
-          setZeroTabIndex = true
-        } else {
-          itemContent.setAttribute('tabindex', '-1')
-        }
```

Comment:
✅ This makes sense to me

When `updateTabIndices` was called after an item was checked the following would happen"
The First item would `setZeroTabIndex = true`
All other items would  `itemContent.setAttribute('tabindex', '-1')`
^ therefore if the las item focused before calling this functio was not the first item all focus would not be possible.

Reply:
Yes, that makes sense! 🎉